joomla/joomla-cms
# Guided tours removal of tour permissions

Pull Request as a replacement for #40208 and #40220. It compiles some of the corrections made and simplifies the code.

### Summary of Changes

Individual tours have permissions in the original Guided Tours PR.
However, tours don't really require such granularity in most cases and the issues linked to having tours in the asset table outweighs the need for such permissions.
This PR removes permissions for individual tours.

### Testing Instructions

Ideally, create several user profiles with limited permissions and make sure those having limited permissions can't access the functionality they are denied.

### Actual result BEFORE applying this Pull Request

Tours can have individual permissions

### Expected result AFTER applying this Pull Request

Tours and steps only get the general permissions of the component.

### Link to documentations
Please select:
- [x] Documentation link for docs.joomla.org: need to remove the permissions section for tours at [Guided Tours: New or Edit Tour](https://docs.joomla.org/Help4.x:Guided_Tours:_New_or_Edit_Tour)
